CreateAccount with Adminkey functionality

Jared-Schimpf opened the pull request:
Works on my machine. This is built with the assumptions I mentioned in the github: that AdminKey is an IT attribute. Check the Database for valid Admin keys to test account creation.

Jared-Schimpf committed:
CreateAccount with Adminkey functionality

Works on my machine. This is built with the assumptions I mentioned in the github: that AdminKey is an IT attribute.
Check the Database for valid Admin keys to test account creation.

Delinquent-Pointer reviewed (approved):
😏🫡Roger works on your machine

Jared-Schimpf committed:
Prevent username overlap

Checks if input username already exists in the DB before trying to create an account.

Also reformatted the IT user handling a little.

Jared-Schimpf commented:
Merge the other branch that contains both features.
